app/views/data-analytics/data-entry-filter-controls.html

```diff
@@ -129,7 +129,8 @@
                   k-options="FormsTypesOptions"
             ></select>
 					<div class="input-group-addon" style="cursor: pointer;" ng-click="selectAllForms()">
-						<span ng-hide="isBusy">All</span>
```

Comment:
the check for is busy is important as it will allow selection only when the controls are ready i.e. control list has been populated.